pydantic/import_helper.py

```diff
@@ -0,0 +1,382 @@
+    'pydantic.schema.field_class_to_schema': V2MigrationMovedWarning('TODO'),
+    'pydantic.schema.json_scheme': V2MigrationMovedWarning('TODO'),
+    'pydantic.schema.add_field_type_to_schema': V2MigrationMovedWarning('TODO'),
+    'pydantic.schema.get_schema_ref': V2MigrationMovedWarning('TODO'),
+    'pydantic.schema.field_singleton_schema': V2MigrationMovedWarning('TODO'),
+    'pydantic.schema.multitypes_literal_field_for_schema': V2MigrationMovedWarning('TODO'),
+    'pydantic.schema.encode_default': V2MigrationMovedWarning('TODO'),
+    'pydantic.schema.get_annotation_from_field_info': V2MigrationMovedWarning('TODO'),
+    'pydantic.schema.get_annotation_with_constraints': V2MigrationMovedWarning('TODO'),
+    'pydantic.schema.normalize_name': V2MigrationMovedWarning('TODO'),
+    'pydantic.schema.SkipField': V2MigrationMovedWarning('TODO'),
+    # pydantic.tools
+    'pydantic.tools.parse_file_as': V2MigrationMovedWarning('TODO'),
+    'pydantic.tools.parse_raw_as': V2MigrationMovedWarning('TODO'),
+    # pydantic.types
+    'pydantic.types.NoneStr': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.NoneBytes': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.StrBytes': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.NoneStrBytes': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.OptionalInt': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.OptionalIntFloat': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.OptionalIntFloatDecimal': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.OptionalDate': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.StrIntFloat': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.ConstrainedNumberMeta': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.ConstrainedInt': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.ConstrainedFloat': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.ConstrainedBytes': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.ConstrainedStr': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.ConstrainedSet': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.ConstrainedFrozenSet': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.ConstrainedList': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.PyObject': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.ConstrainedDecimal': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.FilePath': V2MigrationMovedWarning('TODO'),  # Moved to PathType??
+    'pydantic.types.JsonWrapper': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.JsonMeta': V2MigrationMovedWarning('TODO'),
+    'pydantic.types.ConstrainedDate': V2MigrationMovedWarning('TODO'),
+    # pydantic.typing # TODO
+    # pydantic.utils # TODO
+    # pydantic.validators # TODO
+    # pydantic.version
+    'pydantic.version.compiled': V2MigrationMovedWarning('TODO'),
+    # Case: Moved path
+    'pydantic.error_wrappers.ValidationError': (
```

Comment:
for the moved cases we should raise a warning, but return the new correct thing.